ffi/capi/src/fixed_decimal.rs

```diff
@@ -49,77 +49,94 @@ pub mod ffi {
         MultiplesOf25,
     }
 
-    /// Mode used in a rounding operation.
-    #[diplomat::rust_link(fixed_decimal::RoundingMode, Enum)]
-    #[diplomat::enum_convert(fixed_decimal::RoundingMode, needs_wildcard)]
-    pub enum FixedDecimalRoundingMode {
-        Ceil,
+    /// Mode used in a rounding operation for unsigned numbers.
+    #[diplomat::rust_link(fixed_decimal::UnsignedRoundingMode, Enum)]
+    #[diplomat::enum_convert(fixed_decimal::UnsignedRoundingMode, needs_wildcard)]
+    pub enum FixedDecimalUnsignedRoundingMode {
         Expand,
-        Floor,
         Trunc,
-        HalfCeil,
         HalfExpand,
-        HalfFloor,
         HalfTrunc,
         HalfEven,
     }
 
-    impl FixedDecimal {
-        /// Construct an [`FixedDecimal`] from an integer.
+    /// Mode used in a rounding operation for signed numbers.
+    #[diplomat::rust_link(fixed_decimal::SignedRoundingMode, Enum)]
+    #[diplomat::enum_convert(fixed_decimal::SignedRoundingMode, needs_wildcard)]
+    pub enum FixedDecimalRoundingMode {
```

Comment:
@robertbastian 

How to add the extension `Unsigned(FixedDecimalUnsignedRoundingMode)` in diplomat?

Reply:
You need to flatten it. Did you see this comment? https://github.com/unicode-org/icu4x/issues/5065#issuecomment-2452995367

Reply:
I saw it, but we did not agree on it.

Reply:
Please respond then so we can discuss it?

Reply:
I think you should have 2 enums, and the second enum contains all the variants of the first plus the four extra ones.

Reply:
done